README.md

```diff
@@ -179,6 +179,10 @@ a suggestion is attempted to be highlighted either by the keyboard or mouse. It
 scope, access to an object called `suggestion` which is the current suggestion that is being
 interacted with. A non-selectable suggestion cannot be clicked on, hovered over, or interacted with
 via the keyboard.
+ - `inputHint({query}) {Promise}`: An expression that hints at a suggestion by resolving a
```

Comment:
Instead of making this a totally separate binding, what do you think of making it so that we expect the `sourceFn` to do this? As in, instead of returning what the `sourceFn` does now, the app-maker can include the hint like this:

```
{
  suggestions: [ ... ],
  hint: 'My String'
}
```

That way they don't have to make a totally separate binding in the HTML, which I think is more pleasant API. Of course, we can be backwards compatible and accept both the original data structure, but if we detect that there is the `hint` property on the sourceFn return value, we use it.